app/controllers/admin/accounts_controller.rb

```diff
@@ -14,6 +14,8 @@ def index
     def show
       authorize @account, :show?
 
+      flash.now[:notice] = I18n.t('accounts.instance_actor_flash') if @account.instance_actor?
```

Comment:
Why use `flash` when the note could just be rendered in the template itself?

Reply:
idk, it seemed fitting. Changed it to render the same way, but in the template itself.